docs/TriageProcess.md

```diff
@@ -0,0 +1,50 @@
+Managing a popular GitHub repo with a small team is not an easy task. It requires a good balance between creating new features while handling many investigations and bug fixes associated with existing ones.
+
+During the last couple of years the amount of incoming issues has been constantly growing. While this is a sign of a healthy framework and ecosystem surrounding it, it's becoming harder to react to all those issues.
+To be able to keep up with ever-evolving expectations, we're introducing a set of rules to help us better handle the incoming issues going forward.
+
+## Goals
+The goals of these rules are listed below in priority order:
+- Make it easy to make triage decisions for each issue filed in this repository
+- Be able to easily prioritize issues for each milestone
+- Set proper expectations with customers regarding how issues are going to be handled
+
+## Triage Process
+The feature teams should be able look through every single issue filed against this repository and be able to make a quick call regarding the nature of the issue.
+We will first categorize the issues and further handle these depending on the category the issue is in. The subsections below reprsent these categories and the rules we apply for them during our triage meeting.
+
+### Feature requests
+As soon as we identify an issue represents an ask for a new feature, we will label the issue with the `enhancement` label.
+Most of the time, we will automatically move these issues into the `Next Sprint Planning` milestone for further review during the [next sprint planning meeting]().
+If we think the feature request is not aligned with our goals, we may choose to close it immediately.
+In some situations, however, we may choose to collect more feedback before acting on the issue. In these situations we will move the issue in the `Backlog` so that we can review it during our [release planning meeting]().
```

Comment:
Are you planning to write a separate doc on release planning?

Reply:
Thanks for the reminder. These are supposed to refer to the `Milestone planning process` below. Will update that now.